rules.ini

```diff
@@ -20,6 +20,10 @@ PIXI_JS = (?:^|/)pixi\.js$
 PK3 = \.pk3$ ; idTech3
 RIM = \.rim$ ; BioWare
 RPF = \.rpf$ ; RAGE
+RPGMaker2000[] = (?:^|/)rp[tg]_rt\.exe$
+RPGMaker2000[] = (?:^|/)RPG_RT\.ini$
```

Comment:
you should rename these 2 evidences to `RPGMaker200x` if they are for for both 2000 & 2003

Reply:
Oh, good idea.

Reply:
That's weird, i just made the changes, but some of the errors that are coming up still mention RPG Maker 2000.

Reply:
Oh wait, i think they still say that because "FileDetector.php" decides that the engine is RPG Maker 2000 if it doesn't look like 2003.